src/app/BE/attribute/index.js

```diff
@@ -127,6 +127,8 @@ const Attribute = ({
     <ProgressBar data={data} />
   ) : component === 'radio' || component === 'boolean' ? (
     <Radio.Read data={data} boolean={component === 'boolean'} />
+  ) : component === 'time' ? (
```

Comment:
you can do the same here as above in the ask file
(component === 'text' || component === 'time')

Reply:
Fixed in latest commit